lib/al/Library/Collision/PartsConnector.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#include <math/seadMatrix.h>
+#include "PartsMtxConnector.h"
+
+namespace al {
+class ActorInitInfo;
+class CollisionDirector;
+class CollisionParts;
+class HitInfo;
+class HitSensor;
+class LiveActor;
+class ScreenPointTarget;
+
+class CollisionPartsConnector : public al::MtxConnector {
+private:
+    const al::CollisionParts* mCollisionParts;
+
+public:
+    CollisionPartsConnector();
+    void init(const sead::Matrix34f*, const sead::Matrix34f&, const al::CollisionParts*);
+    void getConnectingSensor() const;
+
+    virtual bool isConnecting() const override;
```

Comment:
```suggestion
    bool isConnecting() const override;
```
If it's overriding, don't repeat `virtual`. Same below.